.idea/JavaScript_course.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@saeidnavaser 
Please use .gitignore file to exclude xml files ??? Id not know from where you have those files ??
When you use them you have to make a change in your code and delete those file from git cache.